setup.cfg

```diff
@@ -18,6 +18,9 @@ exclude = configobj,passlib,cite.py,conf.py
 match = jsondict.py
 ignore = D105, D107, D203, D213
 
+[mypy]
+ignore_missing_imports = True
```

Comment:
`ignore_missing_imports` is enabled so that `mypy` doesn't get mad about missing type annotations in dependencies like `deprecation` or `numpy`.